src/utils/__tests__/ens.test.ts

```diff
@@ -3,31 +3,31 @@ import { validateENS } from '../ens';
 describe('valid names', () => {
   it('domain', () => {
     expect(validateENS('lol.eth')).toMatchInlineSnapshot(`
-      Object {
+      {
```

Comment:
removing the 'Object' as the updated jest version supports a new format for inline snapshots